src/lib/shared/components/ui/Jumbotron/Jumbotron.spec.tsx

```diff
@@ -3,24 +3,58 @@ import { Jumbotron, TEST_IDS } from './Jumbotron';
 
 describe('Jumbotron', () => {
     it('should render', () => {
-        const { getByTestId } = renderWithTheme(<Jumbotron headerText="Test" />);
+        const { getByTestId } = renderWithTheme(
+            <Jumbotron
+                headerText="Test"
+                backgroundProps={{ backgroundImageUrl: 'hi' }}
+            />
+        );
         const container = getByTestId(TEST_IDS.CONTAINER);
         expect(container).toBeVisible();
     });
     it('should render header text', () => {
-        const { getByText } = renderWithTheme(<Jumbotron headerText='Test text' />);
-        const header = getByText(/test text/i);
+        const { getByText } = renderWithTheme(
+            <Jumbotron
+                headerText="Test text"
+                backgroundProps={{ backgroundImageUrl: 'hi' }}
+            />
+        );
+        const header = getByText('Test text');
         expect(header).toBeVisible();
     });
     it('should render subheader text', () => {
-        const { getByText } = renderWithTheme(<Jumbotron headerText='Test text' subHeaderText='Other text' />);
+        const { getByText } = renderWithTheme(
+            <Jumbotron
+                headerText="Test text"
+                subHeaderText="Other text"
+                backgroundProps={{ backgroundImageUrl: 'hi' }}
+            />
+        );
         const header = getByText(/other text/i);
         expect(header).toBeVisible();
     });
-    it('should render call callback fn when button is clicked', () => {
+    it('should use celebration styling when respective prop is true', () => {
+        const { getByTestId } = renderWithTheme(
+            <Jumbotron
+                headerText="Test text"
+                backgroundProps={{ withCelebrationStyling: true }}
+            />
+        );
+        const celebrationBackground = getByTestId(TEST_IDS.CELEBRATION);
+        expect(celebrationBackground).toHaveStyle({
+            background: /linear-gradient.*/,
+        });
```

Comment:
Sorry I missed this before. This definitely works, but I would also be comfortable just testing that the celebration background is visible. We just care that the CelebrationBackground renders in this component. I'd say asserting that it has a particular style is a test case that the Celebration component's unit tests should handle. 

For example, if we decided our celebration UI should just be a solid color moving forward, we wouldn't necessarily expect update to break the Jumbotron tests.